cloudformation-template.json

```diff
@@ -130,7 +130,7 @@
 									"s3:GetObject"
 								],
 								"Resource": [
-									"arn:aws:s3:::*"
+									{"Fn::Sub" : "arn:aws:s3:::${UploadsS3Bucket}"}
```

Comment:
Haven't actually tested this yet... Will update with a comment as soon as I do this.

Reply:
Ok, it does get the correct value for the bucket :)

Reply:
mmm does this need a `*` at the end? I had assumed that a `GetObject` call for `s3://test-bucket/path/to/file.jpg` will need have to match a ARN like `arn:aws:s3:::test-bucket/*`

Reply:
Sure! Totally missed this 😅